Tools/Scripts/libraries/resultsdbpy/resultsdbpy/view/templates/constants.js

```diff
@@ -25,5 +25,8 @@ const XCODE_CLOUD_SUITES = [{% for suite in XcodeCloud %}
     '{{ suite }}',
 {% endfor %}];
 const DEFAULT_ARCHITECTURE = {{ default_architecture }};
+const TESTS_LIMITS = JSON.parse('{{ tests_limits|safe }}');
```

Comment:
Is this a valid JSON string?

Reply:
It's a template file. So it _will_ be valid JSON when we `json.encode` a dictionary and pass it to the template